### PR TITLE
update hash syntax in readme

### PR DESCRIPTION
**Pull request by Elektron1c97**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ flash.assert_flash_is_present(:notice, 'Article saved') # verify that a given fl
 ```ruby
 table = CornerStones::Table.new('.articles')
 table.rows # returns an array of rows. Each row is represented as a row object.
-table.row('Title' => 'Management') # returns the row object for the row with 'Management' in the 'Title' column
+table.row(Title: 'Management') # returns the row object for the row with 'Management' in the 'Title' column
```
Comment (senny) [inline README.md]: Can we keep the current Hash style for `String => String` pairs? In this case the API specifies "<Label> => <Value>" which should both be strings.

File: README.md [hunk 2]
```diff
@@ -62,8 +62,8 @@ table = CornerStones::Table.new('.articles')
 table.extend(CornerStones::Table::SelectableRows)
 table.extend(CornerStones::Table::DeletableRows)
 
-table.row('Created at' => '01.12.2001').select # select the row, which has '01.12.2001' in the 'Created at' column
-table.row('ID' => '9').delete # delete the row, which contains '9' in the 'ID' column
+table.row('Created at': '01.12.2001').select # select the row, which has '01.12.2001' in the 'Created at' column
```
Comment (senny) [inline README.md]: same as https://github.com/senny/corner_stones/pull/40/files#r42628039

File: README.md [hunk 3]
```diff
@@ -62,8 +62,8 @@ table = CornerStones::Table.new('.articles')
...
+table.row(ID: '9').delete # delete the row, which contains '9' in the 'ID' column
```
Comment (senny) [inline README.md]: same as https://github.com/senny/corner_stones/pull/40/files#r42628039

File: README.md [hunk 4]
```diff
@@ -71,12 +71,12 @@ table.values("ID") # => ["1", "7", "9"]
 
 ```ruby
 form = CornerStones::Form.new('.new-article')
-form.fill_in_with('Title' => 'Some Article', 'Author' => 'C. J.') # fill out the form
+form.fill_in_with(Title: 'Some Article', Author: 'C. J.') # fill out the form
```
Comment (senny) [inline README.md]: same as https://github.com/senny/corner_stones/pull/40/files#r42628039

File: README.md (hunk 1 above)
Comment (Elektron1c97) [inline README.md]: @senny of course ;)

File: README.md [hunk 5]
```diff
@@ -63,20 +63,20 @@ table.extend(CornerStones::Table::SelectableRows)
 table.extend(CornerStones::Table::DeletableRows)
 
 table.row('Created at' => '01.12.2001').select # select the row, which has '01.12.2001' in the 'Created at' column
-table.row('ID' => '9').delete # delete the row, which contains '9' in the 'ID' column
-table.values("ID") # => ["1", "7", "9"]
+table.row('ID' '9').delete # delete the row, which contains '9' in the 'ID' column
```
Comment (senny) [inline README.md]: this introduces a syntax error...

File: README.md (hunk 4 above)
Comment (senny) [inline README.md]: @Elektron1c97 you missed this one.

File: README.md (hunk 5 above)
Comment (Elektron1c97) [inline README.md]: @senny Sorry for mistyping..

**Comment by senny** [comment]
@Elektron1c97 can you squash all commits into a single one (`rebase -i`)? and fix the last occurrence (https://github.com/senny/corner_stones/pull/40/files#r42632549). Then I can merge it :hamster:

**Pull request merged**